src/pages/HomePage/Componets/OurServices/OurServicesList/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+
+import { OurServicesListItem } from "../../index";
+
+const OurServicesList = () => {
+	const list = [
+		{
+			id: 0,
+			title: "Development",
+			service: {
+				service_list0: "Website",
+				service_list1: "Mobile apps",
+				service_list2: "CMS",
+				service_list3: "Web apps",
+				service_list4: "eCommerce",
+				service_list5: "Aps"
+			}
+		},
+		{
+			id: 1,
+			title: "Design",
+			service: {
+				service_list0: "UX/UI",
+				service_list1: "Mobile apps",
+				service_list2: "Packaging",
+				service_list3: "Brand identify",
+				service_list4: "Print",
+				service_list5: "Motion design"
+			}
+		},
+		{
+			id: 2,
+			title: "Marketing",
+			service: {
+				service_list0: "Strategy",
+				service_list1: "Naming",
+				service_list2: "Photo & Video",
+				service_list3: "Copy writing",
+				service_list4: "Trademarks"
+			}
+		}
+	];
+
+	const [servicesList, setServicesList] = useState([]);
```

Comment:
This is redundant. Since we are not using updating state anywhere, we can declare `list` as constant and use it for mapping.

```
const LIST = [{...}]

const OurServicesList = () => <div>{LIST.map(...
```

Useful info: https://beta.reactjs.org/learn/state-a-components-memory